osu.Game.Rulesets.Mania/UI/ManiaColourCode.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Mania.UI
+{
+    public enum ManiaColourCode
+    {
+        Off,
+        On
+    }
```

Comment:
Not sure why this is a thing. What you have there is a glorified `bool`. I don't think there will ever be any more options than these.